public/video-ui/src/actions/VideoActions/getVideo.js

```diff
@@ -0,0 +1,37 @@
+import {fetchVideo} from '../../services/VideosApi';
+
+function requestVideo(id) {
+  return {
+    type:       'VIDEO_GET_REQUEST',
+    id:         id,
+    receivedAt: Date.now()
+  };
+}
+
+function recieveVideo(video) {
+  return {
+    type:       'VIDEO_GET_RECIEVE',
+    video:      video,
+    receivedAt: Date.now()
+  };
+}
+
+function errorRecievingVideo(error) {
+  return {
+    type:       'VIDEO_ERROR',
```

Comment:
This could just be the general `SHOW_ERROR` from below, saves having to implement a load of different error reducers